config/site.yml.tmpl

```diff
@@ -25,6 +25,11 @@ secure_cookies: false
 #
 secret_token: "change-me"
 
+# By default, tell Rails to not serve static assets. The best practice is to
```

Comment:
I think the first line should describe the functionality of this option, regardless of how it's set. Then the true/false values, and the reason for the default, will be on the following lines.